public/app/features/plugins/admin/pages/Library.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+import { PluginList } from '../components/PluginList';
+import { usePlugins } from '../hooks/usePlugins';
+import { Page as PluginPage } from '../components/Page';
+import { Loader } from '../components/Loader';
+import { CatalogTab, getCatalogNavModel } from './nav';
+import { Page } from 'app/core/components/Page/Page';
+
+export default function Library(): JSX.Element | null {
+  const { isLoading, items, installedPlugins } = usePlugins();
+  const styles = useStyles2(getStyles);
+
+  const filteredPlugins = items.filter((plugin) => !!installedPlugins.find((_) => _.id === plugin.slug));
```

Comment:
Looking at this now it feels strange that we get back `installedPlugins` from the hook then we filter them to get the installedPlugins. Maybe we should be doing this in the hook? 🤔